include/CL/sycl/opencl_type.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef TRISYCL_SYCL_OPENCL_TYPE_HPP
+#define TRISYCL_SYCL_OPENCL_TYPE_HPP
+
+/** \file
+    triSYCL wrapper for openCL types
+    Joan DOT Thibault AT ens-rennes DOT fr
+    This file is distributed under the University of Illinois Open Source
+    License. See LICENSE.TXT for details.
+*/
+
+#include <boost/compute.hpp>
+
+namespace cl {
+namespace sycl {
+  typedef struct{
+    ::cl_float3 yo; //yo: spannish word for I
```

Comment:
Make this private?
Then change the struct in a class?
Can this work without any constructor?
Where are the unit tests?